packages/mobile/src/app/saga.ts

```diff
@@ -120,7 +120,9 @@ export function* appRemoteFeatureFlagSaga() {
   try {
     while (true) {
       const flags: RemoteFeatureFlags = yield take(remoteFeatureFlagChannel)
-      Logger.info(TAG, 'Updated feature flags', JSON.stringify(flags))
+      const flagsStr = JSON.stringify(flags)
+      Logger.info(TAG, 'Updated feature flags', flagsStr)
+      ValoraAnalytics.track(FeatureFlagsEvents.feature_flags_loaded, { flags: flagsStr })
```

Comment:
Instead of sending a string in a `flags` property can we just send the `flags` thing directly?

Like this:
```
ValoraAnalytics.track(FeatureFlagsEvents.feature_flags_loaded, { flags })
```

This way we'll be able to query using SQL, otherwise it will be quite difficult to query and know which group a user belongs to